aim/cli/convert/utils.py

```diff
@@ -0,0 +1,254 @@
+import json
+import os
+
+import click
+
+from aim import Image, Run
+
+
+def parse_tf_events(tf_logs, repo_inst, flat=False):
+    """
+    This function scans and collects records from TF event files.
+
+    Creates and uses cache file "tf_logs_cache" in the repo dir
+    to track previously processed files and values
+
+    Currently, supports the following TF plugins
+        - scalar
+        - image
+
+    To make conversion process smooth ensure that your directory has the following structure
+
+    your_provided_log_dir/
+        ├> run_1/
+        │    ├> <tf_events_file_1>
+        │    └> <tf_events_file_2>
+        ├> group_1/
+        │    ├> <tf_events_file_3> (THIS EVNET WILL BE IGNORED)
+        │    ├> run_2/
+        │    │    ├> train/
+        │    │    │    ├> <tf_events_file_4>
+        │    │    │    └> <tf_events_file_5>
+        │    │    ├> validate/
+        │    │    │    ├> <tf_events_file_6>
+        │    │    │    └> <tf_events_file_7>
+        │    │    ├> <tf_events_file_8>
+        │    │    └> <tf_events_file_9>
+        │    └> run_3/
+        │        ├> <tf_events_file_10>
+        │        └> <tf_events_file_11>
+        ├> <tf_events_file_12> (THIS EVNET WILL BE IGNORED)
+        └> <tf_events_file_13> (THIS EVNET WILL BE IGNORED)
+
+    From the hierarchy example above you can see that
+    "your_provided_log_dir/group1/tf_events_file_3",
+    "your_provided_log_dir/tf_events_file_12" and
+    "your_provided_log_dir/tf_events_file_13"
+    event files will be ignored since the converter threats them as unorganized event files.
```

Comment:
In contrast to those mentioned, `tf_events_file_8` is _not_ ignored, right?
Can you please elaborate in comments what happens to that event file.
It takes default context, I guess?

Reply:
You're right, its empty context - I'll add it to docstring